api/api/v1/error.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package clutch.api.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/api/v1;apiv1";
+
+import "google/rpc/status.proto";
+
+// Any error information beyond code and status should be included here and added to the error in the status details
+// field. The frontend knows how to render all of the fields in a user-friendly way. If there is extremely verbose error
+// information, consider adding it using a different type, e.g. from the errdetails package. Any details not using this
+// type will still be accessible to the user in a raw format.
+message ErrorDetails {
+  // If there are any underlying errors that were being wrapped, they are presented here.
+  repeated google.rpc.Status wrapped = 1;
```

Comment:
I dont think I can see the full picture, but how is a list of status useful to the FE without mappings to more descriptive error messages? Or is that part coming next?

Reply:
Oh i see now https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto#L28-L47

Reply:
I was thinking this was just status code at first

Reply:
i don't follow the question exactly. each status has it's own message and code. this field allows us to render the highlighted region on the frontend:
![image](https://user-images.githubusercontent.com/4712430/110016898-2f709c00-7ceb-11eb-8a85-37bc7bb373ed.png)